### PR TITLE
(editFormName) edit the name of the form on the design page

### PR DESCRIPTION
**Pull request by raphaeldieu**

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  url = "<%= ENV['URL'] %>" || 'http://localhost:3000';
+  
+
+  $scope.changeFormName = function(newName) {
+    $scope.form = {
+      name: newName
+    };
+    $http.put(url + '/forms/' + $routeParams.id ,$scope.form).success(function(response) {
+
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb:11]: What does it do after success?

File: app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    };
+    $http.put(url + '/forms/' + $routeParams.id ,$scope.form).success(function(response) {
+       
+    }).error(function(response){
+      alert("write something you douche");
+    });
+  }
+
+  $scope.ok = function(newName) {
+    $rootScope.$emit('formName',[newName]);
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb:18]: If you can avoid using this, please do as it takes a lot of memory and is prone to errors

**Comment by h8rry** [comment]
@raphaeldieu please review

File: app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb (hunk 1 above)
Comment (f3r) [inline app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb:11]: you guys should notifiy the user when there is an action done, for example, you can use [bootstrap alerts](http://getbootstrap.com/javascript/#alerts) with an auto-dismiss after 5 secs?

Comment (raphaeldieu) [inline app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb:11]: The behaviour we want is that when you change the name in the modal and click on 'change' the user can see the title of the form changed so there is no need for a confirmation pop up.

File: app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb (hunk 2 above)
Comment (raphaeldieu) [inline app/assets/javascripts/angular/controller/forms/modalChangeNameCtrl.js.erb:18]: well noted, I saw with fer and will keep it like this for now. Fer will teach me the right way to do it later on for my personal grow as a developer :+1:

**Comment by raphaeldieu** [comment]
please check my comments

**Pull request merged**